rules-tests/test.js

```diff
@@ -0,0 +1,165 @@
+const firebase = require('@firebase/testing');
+const fs = require('fs');
+
+/* eslint-env node, mocha */
+
+/*
+ * ============
+ *    Setup
+ * ============
+ */
+const projectId = 'cv19assist-firestore-testing';
```

Comment:
Should this be hard coded?  Also, are we dependent on the project ID?

Reply:
This can be pretty much anything for tests, no? That actually brings up a question I have: You have a json schema for the db. Where is that schema enforced? At the firestore level, kind of like the rules? If not, by convention in the app?

Reply:
@prescottprue do you have any insights on the project ID here?

Re: schema.  It's not enforced anywhere, and not even guaranteed to be valid JSON Schema.  I had started with a very basic text file and then only recently started moving to the JSON Schema-based version.  Now with the automated testing implemented, I do wonder if those should be used in the tests...  It would be a nice to have at this stage, but something to consider.

Reply:
I usually use `process.env.GCLOUD_PROJECT || 'test-project'` so that you can dynamically set the project if you are already using a certain namespace (nice if you wanted to be in the same namespace as other tests) or just have it fall back to a string that can be whatever since we are pointed to the emulators

Reply:
Added in the `process.env.GCLOUD_PROJECT` line.